ament_tools/verbs/build/cli.py

```diff
@@ -114,13 +114,13 @@ def prepare_arguments(parser, args):
         help='End with a particular package',
     )
     parser.add_argument(
-        '--only-package',
-        '--only',
-        help='Only process a particular package, implies --start-with <pkg> and --end-with <pkg>'
+        '--only-packages',
+        nargs='*', default=[],
+        help='Only process a particular set of packages'
```

Comment:
`'List of packages to process'` would be more consistent with the --skip-packages help (and makes it clear it's not a flag)